core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-create/container-create.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'dot-container-create',
+    templateUrl: './container-create.component.html',
+    styleUrls: ['./container-create.component.css']
```

Comment:
change to scss (I know the docs says we use scss)

Reply:
On it...